src/ui/VoiceMessageInput/types.ts

```diff
@@ -0,0 +1,7 @@
+export const VoiceMessageInputStatus = {
+  READY_TO_RECORD: 'READY_TO_RECORD',
+  RECORDING: 'RECORDING',
+  READY_TO_PLAY: 'READY_TO_PLAY',
+  PLAYING: 'PLAYING',
+} as const;
+export type VoiceMessageInputStatus = typeof VoiceMessageInputStatus[keyof typeof VoiceMessageInputStatus];
```

Comment:
@HoonBaek 
export type VoiceMessageInputStatus > VoiceMessageInputStatusTypes is better, can you refactor on this branch?